app/js/daemonAPI.js

```diff
@@ -36,7 +36,11 @@ function sendCall(url, type, args, callback) {
 		url: url,
 		type: type,
 		success: function(responseData, textStatus, jqXHR) {
-			callback(null, JSON.parse(responseData), textStatus, jqXHR);
+			try {
+				callback(null, JSON.parse(responseData), textStatus, jqXHR);
+			} catch(e) {
+				console.error(e, 'Ajax response was: ' + responseData);
```

Comment:
Catches improperly constructed JSONs that JSON.parse would normally return a weird error on